src/main/java/cat/udl/eps/softarch/demo/handler/RequestEventHandler.java

```diff
@@ -34,12 +34,28 @@ public void handleRequestPreSave(Request newRequest) {
         assert newRequest.getId() != null;
         Optional<Request> requests = requestRepository.findById(newRequest.getId());
 
+        if(requests.isPresent()){
+            throw new ForbiddenException();
+        }
+
+        /* ESTO NO SIRVE PARA NADA PERO LO DEJO PARA VER LA MIERDA QUE IMPLEMENTO SARA CUANDO IBA DE EXTASIS HASTA ARRIBA
```

Comment:
Please, remove commented section from the final version that will go into the main branch